antora-ui-camel/src/css/frontpage.css

```diff
@@ -188,6 +188,10 @@ section.frontpage h6 {
   text-transform: uppercase;
 }
 
+section.frontpage h2 {
+  font-size: 22px;
```

Comment:
We should either use a different heading in the markup (like h3 instead of h2), or change the `px` to `rem` here.

Reply:
I think It will be better to use different heading format.